project-reports/2024/2024-annual-Hyperledger-Cacti.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: 2024 Annual Review Hyperledger Cacti
+parent: 2024
+grand_parent: Project Updates
+---
+
+# Project Health
+
+[Hyperledger Cacti LFX Insights page](https://insights-v2.lfx.linuxfoundation.org/cactus/trends)
+
+1. We've been (mostly) steadily adding new contributors each month with a quick break during the Summer.
+2. The pull request lead time has been reduced to 15 days which is great news, previously this was well above 20, meaning that we've had a 60% decrease overall. 
+3. What we especially needed to improve on with pull request lead times was the waiting for review state, in which, pull requests should not spend more than 48 hours according to our self-imposed limit and we are closing in on hitting this milestone with the waiting for review state lasting 2.76 days on average.
+4. We've resolved ~670 issues but 1070 were opened. This is can be seen through many different lenses. One of them is that more than 30% of the currently open issues are "good first" issues that we mostly keep open as bait for new contributions in order to grow the community.
```

Comment:
```suggestion
4. We've resolved ~670 issues but 1070 were opened. This can be seen through many different lenses. One of them is that more than 30% of the currently open issues are "good first" issues that we mostly keep open as bait for new contributions in order to grow the community.
```

Reply:
@tkuhrt Thank you very much for the suggestion, I applied it!